docs/core/compatibility/containers/8.0/aspnet-port.md

```diff
@@ -9,6 +9,8 @@ The default ASP.NET Core port configured in .NET container images has been updat
 
 We also added the new `ASPNETCORE_HTTP_PORTS` environment variable as a simpler alternative to `ASPNETCORE_URLS`. The new variable expects a semicolon delimited list of ports numbers, while the older variable expects a more complicated syntax.
 
+Apps built using the older <xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder?displayProperty=nameWithType> API won't respect the new `ASPNETCORE_HTTP_PORTS` environment variable. And now that `ASPNETCORE_URLS` is no longer set automatically, they'll switch to use a default port of 5000.
```

Comment:
```suggestion
Apps built using the older <xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder?displayProperty=nameWithType> API won't respect the new `ASPNETCORE_HTTP_PORTS` environment variable. And now that `ASPNETCORE_URLS` is no longer set automatically, they'll switch to use a default port of 5000, which is not accessible outside the container.
```

Reply:
Is this what you were going for @tmds?

Reply:
witch to use a default port of 5000 **on localhost**

Reply:
Is it even useful to say which port it's using?  I wonder if we should just explain that it will only listen on localhost and will therefore not be functional.

Reply:
I'd mention the port that is bound, but you could leave it out.

This is how ASP.NET Core prints the address:
```
Now listening on: http://localhost:5000
```